src/demo/README.md

```diff
@@ -0,0 +1,19 @@
+# JS SDK Demo App
```

Comment:
Not sure why this has paragraphs with newlines - it sort of breaks the Markdown on e.g. small screens.